hasql.cabal

```diff
@@ -74,7 +74,7 @@ library
     -- parsing:
     attoparsec >= 0.10 && < 0.14,
     -- database:
-    postgresql-binary >= 0.9.1.1 && < 0.10,
+    postgresql-binary >= 0.10 && < 1,
```

Comment:
There's no guarantee that the next major-version bump won't be API-breaking even before `1`. So it needs to be restricted to `0.11`.